docs/source/dataframe-design.rst

```diff
@@ -110,4 +160,6 @@ divided.  In this case ``.divisions`` will be all ``None``:
    >>> df.divisions
    [None, None, None, None, None]
 
-In these cases any operation that requires a cleanly partitioned dataframe with known divisions will have to perform a sort.  This can generally achieved by calling ``df.set_index(...)``.
+In these cases any operation that requires a cleanly partitioned dataframe with
+known divisions will have to perform a sort.  This can generally achieved by
+calling ``df.set_index(...)``.
```

Comment:
It would be useful to somewhere have suggestions on how to create categoricals, perhaps pointing both to the categorize method and to the `astype('category')` method..